lib/private/files/objectstore/swift.php

```
@@ -77,7 +77,11 @@ protected function init() {
 			$serviceName = $this->params['serviceName'];
 		}
 
-		$this->objectStoreService = $this->client->objectStoreService($serviceName, $this->params['region']);
+		$urlType = null;
+		if ($this->params['urlType']) {
```

and have urltype default to  `internal` if that's the default.
Else this means that every existing config out there will have to be changed.

If you think the value must be mandatory in the config, might want to throw an exception.